src/java/src/main/java/ai/onnxruntime/genai/Tensor.java

```diff
@@ -7,6 +7,7 @@
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 
+/** Wraps ORT native Tensor. */
```

Comment:
In need of a better description.

Reply:
Really just a Tensor. It might not necessarily wrap an ORT tensor in the future.

Reply:
Should I change the description to something else?

Reply:
Maybe just "Currently wraps an ORT Tensor" since that implies it could change. But I'm not sure if we should say it's an ORT Tensor as that's an implementation detail. Up to you.

Reply:
"Currently wraps an ORT Tensor" feels right to me. Thanks!